app/tasks/schedule.php

```diff
@@ -0,0 +1,155 @@
+            $functions[$document['scheduleId']] = $document;
+            $count++;
+        }
+    }
+
+    $createQueue();
+    $lastUpdate =  DateTime::addSeconds(new \DateTime(), -FUNCTION_VALIDATION_TIMER);
+
+    Co\run(
+        function () use ($removeFromQueue, $createQueue, $dbForConsole, &$functions, &$queue, &$lastUpdate) {
+            Timer::tick(FUNCTION_VALIDATION_TIMER * 1000, function () use ($removeFromQueue, $createQueue, $dbForConsole, &$functions, &$queue, &$lastUpdate) {
+                $time = DateTime::now();
+                $count = 0;
+                $limit = 50;
+                $sum = $limit;
+
+                Console::info("Update proc run at: $time last update was at $lastUpdate");
+                /**
+                 * Updating functions list from DB.
+                 */
+                while ($sum === $limit) {
+                    $results = $dbForConsole->find('schedules', [
+                        Query::equal('region', [App::getEnv('_APP_REGION')]),
+                        Query::equal('type', ['function']),
+                        Query::greaterThan('scheduleUpdatedAt', $lastUpdate),
+                        Query::limit($limit)
+                    ]);
+                    $sum = count($results);
+                    foreach ($results as $document) {
+                        $org = isset($functions[$document['scheduleId']]) ? strtotime($functions[$document['scheduleId']]['scheduleUpdatedAt']) : null;
+                        $new = strtotime($document['scheduleUpdatedAt']);
+                        if ($document['active'] === false) {
+                            Console::error("Removing:  {$document['scheduleId']}");
+                            unset($functions[$document['scheduleId']]);
+                        } elseif ($new > $org) {
+                            Console::error("Updating:  {$document['scheduleId']}");
+                            $functions[$document['scheduleId']] =  $document;
+                        }
+                        $removeFromQueue($document['scheduleId']);
+                        $count++;
+                    }
+                }
+
+                $lastUpdate = DateTime::now();
+                $createQueue();
+            });
+
+            Timer::tick(FUNCTION_ENQUEUE_TIMER * 1000, function () use ($dbForConsole, &$functions, &$queue) {
+                $time = DateTime::now();
+                $timeFrame =  DateTime::addSeconds(new \DateTime(), ENQUEUE_TIME_FRAME); /** 5 min */
+                $now = (new \DateTime())->format('Y-m-d H:i:00.000');
+
+                Console::info("Enqueue proc run at: $time");
+
+                /**
+                 * Lopping time slots
+                 */
+                foreach ($queue as $slot => $schedule) {
+                    if ($now === $slot) {
+                        foreach ($schedule as $function) {
+                        /**
+                         * Enqueue function
+                         */
```

Comment:
Indentation is wrong. Please use a formatter, it will help us have shorter iteration loops on reviews.